ibis/bigquery/compiler.py

```diff
@@ -454,6 +454,21 @@ def compiles_strftime(translator, expr):
     )
 
 
+@compiles(ops.StringToTimestamp)
+def compiles_string_to_timestamp(translator, expr):
+    arg, format_string, timezone_arg = expr.op().args
+    fmt_string = translator.translate(format_string)
+    arg_formatted = translator.translate(arg)
+    if isinstance(timezone_arg, ir.StringValue):
```

Comment:
Instead of checking the type of `timezone_arg`, just check `timezone_arg is not None`. Ibis's argument validation will have converted the argument to a `StringValue` if it's not `None`.